Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -40,13 +65,3 @@ When in doubt, submit any issue through our [Triage Incident Template](https://g
 
 Create a process flow diagram to compliment what is being said in a visaul manner
```

Comment:
Typo we missed earlier: `visaul` -> `visual`

Reply:
@cvalarida I added a "suggested commit" to this